src/WebJobs.Script/Host/WorkerFunctionMetadataProvider.cs

```diff
@@ -226,18 +252,6 @@ internal static FunctionMetadata ValidateBindings(IEnumerable<string> rawBinding
             return function;
         }
 
-        private bool IsDefaultIndexingRequired(IEnumerable<RawFunctionMetadata> functions)
-        {
-            foreach (RawFunctionMetadata function in functions)
-            {
-                if (function.UseDefaultMetadataIndexing == true)
-                {
-                    return true;
-                }
-            }
-            return false;
-        }
-
         private bool IsNullOrEmpty(IEnumerable<RawFunctionMetadata> functions)
```

Comment:
If you want, this can be removed and replaced with `functions?.Any() != true` in all places.